src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -60,7 +60,7 @@ public AdaptedPerson(ReadOnlyPerson source) {
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```

Comment:
call `toString()` inside `getAddress()` and return the address string from `getAddress()`, can?